docs-www/src/styles/carousel.scss

```diff
@@ -0,0 +1,3 @@
+h2 {
```

Comment:
`carousel.scss`?

I guess this file should be either renamed (with updating its content or not) or removed.